elastic/kibana
# [ResponseOps][Alerting] Update Kibana Alerting scaling metrics to be running counts

Resolves https://github.com/elastic/kibana/issues/145289

## Summary

This pr removed a field and fixes a bug with the previous metrics and keeps a running count for the life of the kibana. 


### Checklist



### To verify

- Create a rule and hit the `/internal/task_manager/_background_task_utilization` api
- Verify that the counts are increasing
